docs/source/tags/timeseries.md

```diff
@@ -61,8 +63,10 @@ Channel tag can be used to label time series data
 </View>
 ```
 ### Example
+
+Labeling configuration for time series data stored in the task field `ts` in Label Studio JSON format. The time field is stored as a date in the `timeformat` field and formatted as a full date on the plot (by default).
+
 ```html
-<!--Labeling configuration for time series data stored in the task field `ts` in Label Studio JSON format. The time field is stored as a date in the `timeformat` field and formatted as a full date on the plot (by default). -->
```

Comment:
Only this sentence ends with a period. Make sure all sentences end with a period (`.`) for completeness.